analysis_options.yaml

```diff
@@ -28,6 +28,8 @@ analyzer:
     missing_return: warning
     # allow having TODOs in the code
     todo: ignore
+    #
```

Comment:
missing comment here

Reply:
Created https://github.com/flutter/flutter/pull/25995 to address this.